Bottom: 4de6d8d33b0e95173e59bbcfe4a016fa20735d3f
Top:    3bcf8ea4d5622c29b01514941bd7292939962e06
Author: Ben Greear <greearb@candelatech.com>
Date:   2016-02-04 15:24:51 -0800

ath10k: print baud & pin for debug uart.

Printing the baud and pin can help the user verify how
the firmware is being configured.


---

diff --git a/drivers/net/wireless/ath/ath10k/core.c b/drivers/net/wireless/ath/ath10k/core.c
index d294f88..6412b35 100644
--- a/drivers/net/wireless/ath/ath10k/core.c
+++ b/drivers/net/wireless/ath/ath10k/core.c
@@ -1437,7 +1437,8 @@ static int ath10k_init_uart(struct ath10k *ar)
 		return ret;
 	}
 
-	ath10k_info(ar, "UART prints enabled\n");
+	ath10k_info(ar, "UART prints enabled: 19200, tx-pin: %d\n",
+		    ar->hw_params.uart_pin);
 	return 0;
 }
